.github/docker-node/10/amazonlinux2/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM amazonlinux:2
+
+ENV NODE_VERSION 10.24.1
+
+# centos needs the user to be root; sudo doesn't work.
+USER root
+# yum returns non-zero exit code (100) if packages available for update
+RUN yum -y check-update || echo "packages available for update"
+
+# install software required for this OS
+RUN yum -y install \
+    tar \
+    gzip
+
+# install nvm
+ENV NVM_DIR /root/.nvm
+
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.1/install.sh | bash
```

Comment:
curl-bash is a little bit frowned on now due to security, I looked a little and not seeing any obvious alternative/manual install method that the nvm project supports -- I will note this as a follow-on task to address.